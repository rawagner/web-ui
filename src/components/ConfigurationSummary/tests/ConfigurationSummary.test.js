import React from 'react';
import { shallow } from 'enzyme';

import { ConfigurationSummary } from '..';

import { default as ConfigurationSummaryFixture } from '../fixtures/ConfigurationSummary.fixture';

const testConfigurationSummary = () => <ConfigurationSummary {...ConfigurationSummaryFixture.props} />;

describe('<ConfigurationSummary />', () => {
  it('renders correctly', () => {
    const component = shallow(testConfigurationSummary());
    expect(component).toMatchSnapshot();
  });
});
