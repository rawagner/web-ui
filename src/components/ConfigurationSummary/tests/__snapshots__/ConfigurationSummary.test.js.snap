// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConfigurationSummary /> renders correctly 1`] = `
<dl
  className="kubevirt-configuration-summary"
>
  <dt>
    Operating System
  </dt>
  <dd>
    ---
  </dd>
  <dt>
    Flavor
  </dt>
  <dd>
    small - 2 CPU, 2G Memory
  </dd>
  <dt>
    Workload Profile
  </dt>
  <dd>
    generic
  </dd>
  <dt>
    NICs
  </dt>
  <dd>
    <div
      key="eth0"
    >
      eth0
    </div>
  </dd>
  <dt>
    Disks
  </dt>
  <dd>
    <div
      key="rootdisk"
    >
      rootdisk
    </div>
    <div
      key="cloudinitdisk"
    >
      cloudinitdisk - cloud-init disk
    </div>
  </dd>
</dl>
`;
